template/node/src/service.rs

```diff
@@ -385,20 +380,18 @@ pub fn new_light(config: Configuration) -> Result<TaskManager, ServiceError> {
 		on_demand.clone(),
 	));
 
-	let grandpa_block_import = sc_finality_grandpa::light_block_import(
-		client.clone(), backend.clone(), &(client.clone() as Arc<_>),
-		Arc::new(on_demand.checker().clone()) as Arc<_>,
+	let (grandpa_block_import, _) = sc_finality_grandpa::block_import(
+		client.clone(),
+		&(client.clone() as Arc<_>),
+		select_chain.clone(),
 	)?;
 
-	let finality_proof_import = grandpa_block_import.clone();
-	let finality_proof_request_builder =
-		finality_proof_import.create_finality_proof_request_builder();
+	//TODO shouldn't we have the aura block import here at least conditionally?
```

Comment:
@sorpaas Can you please take a look at this too? I don't see where the aura block import is included.

Reply:
Yeah I think it's missed there -- we should have an additional aura block import struct created here and use that for the import queue below.

Reply:
Actually it's more than just that import queue that's broken. Mind if we handle that in a followup and keep this PR about updating Substrate?

I've reported it https://github.com/paritytech/frontier/issues/238